### PR TITLE
MODELIX-560 bulk-model-sync quality of life improvements

### PR DESCRIPTION
**Pull request by mhuster23**
This PR adresses 2 pain points:

- Including multiple modules required listing them individually. As a shortcut they can now be included by specifying a prefix.
- The sync was almost like a black box. This PR adds some progress indicators.

### CODE REVIEW

File: bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 package org.modelix.model.sync.bulk
 
+import mu.KotlinLogging
```
Comment (languitar) [inline bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt:19]: Why was this logging library chosen?

File: docs/global/modules/core/pages/reference/component-bulk-model-sync-gradle.adoc [hunk 2]
```diff
@@ -70,6 +70,10 @@ This means that only a minimal amount of write operations is used to update the
 |`includeModule`
 |String
 |Includes the module specified by the given fully qualified name in the synchronisation process.
+
+|`includeModulesByPrefix`
+|String
+|Includes all modules, whose fully qualified name starts with the given prefix, in the synchronisation process.
```
Comment (languitar) [inline docs/global/modules/core/pages/reference/component-bulk-model-sync-gradle.adoc:76]: We should decide once for all modelix if we speak British or American English ;-)

**Review by languitar — APPROVED** [review]
Looks fine to me. Feel free to close the open threads, which are slight remarks.

As a small side note: this PR addresses two distinct concerns: prefix config and progress logging. I would have preferred two separate PRs for easier reviewing.

File: bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt (hunk 1 above)
Comment (mhuster23) [reply bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt:19]: It's just the one we are already using across the core.

**Pull request merged**